### DISCUSSION TITLE
Multiple banks using GoCardless?

### DISCUSSION THREAD

**Discussion by ronnyandre** [firefly-iii/firefly-iii#8256] [Firefly III] [answered]
I want to import data from multiple banks using GoCardless. Is that possible?

**Comment by JC5** [comment, marked as answer]
Yes it is. Go through the importer's UI for each bank. Save the configuration files. You can reuse those the next time.

**Reply by ronnyandre** [reply]
Thanks, you were right. I tried it yesterday, but it had already cached my previous selection.

**Reply by pedroponte** [reply]
How does the data importer work with cron and multiple banks?
Do I have to run it manually every day or does it do it automatically after the first setup on a daily basis? TIA

**Reply by JC5** [reply]
If you have a configuration file, you can refer to it using a cron job you have to set up yourself.
